scripts/testing/README.md

```diff
@@ -0,0 +1,112 @@
+## Automated testing
+
+### Installation
+
+Atomated testing CLI depends on `gcloud` and `vault` CLIs so you need to install them. 
+
+#### Google cloud CLI
+Follow the [offical installation](https://cloud.google.com/sdk/docs/install) guide. Once installed, run `gcloud auth login` and authorize using SSO. 
+
+#### Vault
+Download and install the Hashicorp Vault client version 1.14.1 or newer or via brew with `brew install vault`
+1. `brew install vault`.
+2. [Generate](https://github.com/elastic/infra/tree/master/docs/vault#github-auth) a github token to authorize in Vault.
+3. Run `vault login --method github` to authorize your Vault cli.
+
+#### Connector service
+Run `make clean install` to generate executable files in `./bin` folder
+
+### Usage
+
+***Note:*** The working directory for `./bin/test-connectors` is `scripts/testing` so when you need to point to a file you should use it as a root directory. 
```

Comment:
It's a bit confusing, can we support absolute paths by chance?

Reply:
This confuses me too. I came up with this solution because Python executable scripts use `./bin` as they root. It's not the best user experience. I will change that.